### ISSUE TITLE
Please help with translations before 5.2 will be released

### ISSUE THREAD

**Issue by juek** [Typesetter/Typesetter#603]
Hey guys, Typesetter CMS 5.2 is due. We have some new features that still [need to be translated](https://github.com/Typesetter/Typesetter/tree/master/include/languages). 

We would prefer pull requests, but of course you can also email me the changed language files to jk&zwnj;(at)&zwnj;grafikrausz&zwnj;(dot)&zwnj;at.

Thanks and, hurry up ;)

**Comment by mahotilo** [comment]
There is only one word `Addon` in a language file.
Should it be Addon or Plugin?
Is this check only for Plugins or it can manage Themes too?

**Comment by mahotilo** [comment]
And...
Can we put the deprecated addons list in a separate file and just include it in gpconfig.php?
Just to keep gpconfig.php short. I think we will extend this list soon.

**Comment by juek** [comment]
> Is this check only for Plugins or it can manage Themes too?

Generally, both themes and plugins are addons, Themes can implement plugin hooks but not vice versa.  

> Just to keep gpconfig.php short.

Admittedly, I'm not exactly happy with the gpconfig.php solution but many devs already know this file. Why keep it short?

**Comment by juek** [comment]
But now as you mention it, we most likely need to move $deprecated_addons somewhere else. I'm almost certain gpconfig.php will not be changed with a core update.

**Comment by mahotilo** [comment]
As admin, I can change any option in gpconfig.php. It is allowed and proposed.
But these lines about Deprecated Addons is not for me. I should not change them.
In my sleepy mind:) 'Short' means without extra lines.

**Comment by mahotilo** [comment]
> I'm almost certain gpconfig.php will not be changed with a core update.

Oh, yes. Especially when you just replace the Include folder to update CMS.

**Comment by juek** [comment]
$deprecated_addons [moved](https://github.com/Typesetter/Typesetter/commit/386dfcc434236bda2b7163294acad90af45f1a51) to /include/deprecated.php

BTW: We're off-topic ;)

**Comment by juek** [comment]
I'd like to keep this issue for translation stuff. Mind if I delete the OT discussion?

**Comment by juek** [comment]
I just deleted some off-topic discussion, nothing offensive ;)

**Comment by juek** [comment]
French is [updated](https://github.com/Typesetter/Typesetter/commit/2e57cf28753c1b577934a1d71be37456764b7b71). Thanks Marcus!

**Comment by florincatalin** [comment]
Please update the Romanian version as well, the old one has a bug that I just corrected and I also improved it a bit. Thank you!

**Comment by juek** [comment]
@florincatalin Where is your updated version? [This one](https://github.com/florincatalin/Typesetter/blob/master/include/languages/ro.main.inc) seems rather old

**Comment by juek** [comment]
@florincatalin Ok, found it. You sent it to [your own fork](https://github.com/florincatalin/Typesetter/pull/3) of Typesetter.

**Comment by juek** [comment]
Romanian is [updated](https://github.com/Typesetter/Typesetter/commit/b40f4d518f09ae83afc6ed56eaf66974efd7f1c7). Thanks Florin!

**Comment by florincatalin** [comment]
Yeah, sorry for mixing them up. It is possible that small corrections will appear later.

**Comment by Hoobie** [comment]
@juek Please find attached a Polish translation:
[pl.main.inc.txt](https://github.com/Typesetter/Typesetter/files/6058482/pl.main.inc.txt)
Please remove the `txt` extension.
Sorry for not opening a PR, I'm just passing it around and I'm too lazy to fork the repo.
Cheers
